config/server.js

```diff
@@ -10,7 +10,7 @@ const config = {
     pgUri: process.env.DATABASE_URL || 'postgres://localhost:5432/concaria'
   },
   habanero: {
-    host: process.env.HABANERO_HOST
+    embedded: (process.env.HABANERO_EMBEDDED === false) ? false : true 
```

Comment:
I will check the type of the env var. I am pretty sure it's a string. :/
`(process.env.HABANERO_EMBEDDED || 'true') === 'true'`